src/core_ocean/mode_forward/mpas_ocn_time_integration_split.F

```diff
@@ -1759,11 +1759,11 @@ subroutine ocn_time_integrator_split(domain, dt)!{{{
                                 .or.lat>- 2.5.and.lat<  2.5 &
                                 .or.lat> 35.0.and.lat< 40.0 &
                                 .or.lat> 55.0.and.lat< 60.0 ) then
-                              do k = 1, config_reset_debugTracers_top_nLayers
+                              do k = minLevelCell(iCell), minLevelCell(iCell)+config_reset_debugTracers_top_nLayers
                                  tracersGroupNew(2,k,iCell) = 2.0_RKIND
                               end do
                            else
-                              do k = 1, config_reset_debugTracers_top_nLayers
+                              do k = minLevelCell(iCell), minLevelCell(iCell)+config_reset_debugTracers_top_nLayers
```

Comment:
The upper limit of this loop should also have the `-1` appended (like line 1750)